### PR TITLE
Updates needed for compatibility with CMEPS master

### PR DESCRIPTION
**Pull request by mvertens**
These changes are needed to get the latest nuopc cap working with CMEPS master.

### CODE REVIEW

File: src/drivers/nuopc/ice_comp_nuopc.F90 [hunk 1]
```diff
@@ -89,6 +89,7 @@ module ice_comp_nuopc
   real(R8)               :: orb_obliq       ! attribute - obliquity in degrees
   real(R8)               :: orb_mvelp       ! attribute - moving vernal equinox longitude
   real(R8)               :: orb_eccen       ! attribute and update-  orbital eccentricity
+  logical                :: masterproc
```
Comment (dabail10) [inline src/drivers/nuopc/ice_comp_nuopc.F90]: I had added mastertask as a logical. Should this be removed here?

File: src/drivers/nuopc/ice_comp_nuopc.F90 [hunk 2]
```diff
@@ -348,14 +357,13 @@ subroutine InitializeRealize(gcomp, importState, exportState, clock, rc)
     !----------------------------------------------------------------------------
     ! Get orbital values
     !----------------------------------------------------------------------------
-
+    masterproc = localPet==master_task
```
Comment (dabail10) [inline src/drivers/nuopc/ice_comp_nuopc.F90]: I prefer the syntax I used for mastertask above.

**Review by dabail10 — CHANGES_REQUESTED** [review]
These look generally fine, but I'm not fond of the masterproc syntax.

File: src/drivers/nuopc/ice_comp_nuopc.F90 (hunk 1 above)
Comment (mvertens) [reply src/drivers/nuopc/ice_comp_nuopc.F90]: I am closing  this PR in terms of a new one that utilizes CDEPS for prescribed ice and removes all references to MCT from the nuopc directory.

**Pull request closed without merging**